app/models/tv_season.rb

```diff
@@ -1,34 +1,32 @@
 class TVSeason
-  def initialize(episode_number, name, air_date, guest_stars, overview, still_path)
-    @episode_number = episode_number
-    @name = name
+  attr_accessor :series, :show_id, :air_date, :name, :overview, :season_id, :poster_path, :season_number, :credit, :episodes
+
+  def initialize(series:, show_id:, air_date:, name:, overview:, season_id:, poster_path:, season_number:, credit:, episodes:)
+    @series = series
+    @show_id = show_id
     @air_date = air_date
-    @guest_stars = guest_stars
+    @name = name
```

Comment:
This class was having a little bit of an identity crisis. It was mixing the ideas of `season` and `episode`.  This refactor separates those concerns while still delivering the goods to the html pages.